db/schema.rb

```diff
@@ -175,6 +175,16 @@
     t.index ["teacher_reference_number"], name: "index_ecp_eligibility_trn"
   end
 
+  create_table "eligible_fe_providers", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.integer "ukprn", null: false
+    t.text "academic_year", null: false
+    t.decimal "max_award_amount", precision: 7, scale: 2
+    t.decimal "lower_award_amount", precision: 7, scale: 2
```

Comment:
I'm not sure we'll need a column for the lower amount as it's always 50% of the higher amount.

Reply:
I've now removed this column